components/lib/crash/README.md

```diff
@@ -92,7 +92,8 @@ CrashReporter(
 ### Sending crash reports to Glean
 
 [Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html) is a new way to collect telemetry by Mozilla. 
-This will record crash counts as a labelled counter with each label corresponding to a specific type of crash (`native_code_crash`, and `unhandled_exception`, currently). The list of collected metrics is available in the [metrics.yaml file](metrics.yaml). 
+This will record crash counts as a labelled counter with each label corresponding to a specific type of crash (`native_code_crash`, and `unhandled_exception`, currently).
```

Comment:
```suggestion
This will record crash counts as a labeled counter with each label corresponding to a specific type of crash (`native_code_crash`, and `unhandled_exception`, currently).
```

(Unfortunate) U.S. English

Reply:
Ouch. Thanks for spotting that!

Reply:
No problem -- it trips me up even as a native English speaker.